packages/ts-fdc3-oms/acme-service/src/acmeService.ts

```diff
@@ -304,45 +306,30 @@ class AcmeService {
             quantity: args.quantity,
         }
 
-        this.glue?.intents.raise({
-            intent: 'NewOrder',
-            context: {
+        this.fdc3?.raiseIntent('NewOrder', {
                 type: 'fdc3.order',
-                data: order,
-            },
-            target: 'reuse',
-        })
+                order,
+            }
+        )
     }
 
     private raiseViewOrderHistory = async (context: any): Promise<void> => {
         if (await this.shouldRaiseIntent()) {
-            this.glue.intents
-                .raise({
-                    intent: ViewOrderHistoryIntent,
-                    context,
-                    target: 'reuse',
-                })
+            this.fdc3?.raiseIntent(ViewOrderHistoryIntent, context)
```

Comment:
I don't think the context is in the right format off the bat - search '                    this.raiseViewOrderHistory' in files and see. You would have to normalize the data:

this.raiseViewOrderHistory({
                        data: {
                            securityId: {
                                ticker: ticker,
                                bbgExchange: exchange,
                            },
                            clientId: '',
                        },
                    })
like remove the data prop in this case